src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector512.cs

```diff
@@ -2670,6 +2670,23 @@ public static void StoreUnsafe<T>(this Vector512<T> source, ref T destination)
             ref byte address = ref Unsafe.As<T, byte>(ref destination);
             Unsafe.WriteUnaligned(ref address, source);
         }
+        /// <summary>
+        /// Stores to lower 256 bits of <paramref name="source"/> to memory destination of <paramref name="destination"/>[<paramref name="elementOffset"/>]
+        /// </summary>
+        /// <typeparam name="T">The type of the elements in the vector.</typeparam>
+        /// <param name="source">The vector that will be stored.</param>
+        /// <param name="destination">The destination to which <paramref name="elementOffset" /> will be added before the vector will be stored.</param>
+        /// <param name="elementOffset">The element offset from <paramref name="destination" /> from which the vector will be stored.</param>
+        /// <remarks>
+        /// Uses double instead of long to get a single instruction instead of storing temps on general porpose register (or stack)
+        /// </remarks>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void StoreLowerUnsafe<T>(this Vector512<T> source, ref T destination, nuint elementOffset = 0)
```

Comment:
Same general question here as on the other. Why this helper rather than simply doing `source.GetLower().StoreUnsafe(ref destination, elementOffset)`

It looks like the right stuff happens for the former already and it avoids the JIT needing to do any inlining for it.